pynamodb/models.pyi

```diff
@@ -127,3 +131,11 @@ class BatchWrite(Generic[_T], ModelContextManager[_T]):
     def __exit__(self, exc_type, exc_val, exc_tb) -> None: ...
     pending_operations: Any
     def commit(self) -> None: ...
+
+class _ModelFuture:
+    _model_cls: Type[_T]
+    _model: Optional[_T]
```

Comment:
so i keep getting mypy errors on `_model_cls: Type[_T]` and `_model: Optional[_T]`: `error: Invalid type "pynamodb.models._T"`. Would y'all happen to know why? @garrettheel @ikonst

Reply:
you need to make `_ModelFixture` a generic class:
```
class _ModelFuture(Generic[_T]):
```

Reply:
thank you! that did it :)